tex/paper.tex

```diff
@@ -329,27 +330,173 @@ \subsection{Description of the Axelrod python package}\label{sec:description-of-
 
 \section{Reproducing previous tournaments}\label{sec:reproducing-previous-tournaments}
 
-S&P 2012 is fully reproducible; results are unstable (e.g. substituting soft_majo for
-hard_majo changes the results).
+Due to the open nature of the library the number of strategies included has
+grown at a fast pace, as can be seen in
+Figure~\ref{fig:number_of_strategies_against_date}. Despite this, due to
+previous research being done in an irreproducible way due to, for example no
+source code, and/or vaguely described strategies; not all previous tournaments
+are yet to be reproduced.
```

Comment:
And unfortunately perhaps some of the early tournaments are not reproducible. Maybe remark that some of the source code for strategies of the early tournaments may be forever lost, and this library aims to prevent that from happening again in the future.